mediathread/settings_shared.py

```diff
@@ -184,7 +184,7 @@ def no_reject(request, reason):
 CORS_ALLOW_METHODS = ('GET',)
 CORS_ALLOW_CREDENTIALS = True
 
-SESSION_COOKIE_SAMESITE = 'None'
+DCS_SESSION_COOKIE_SAMESITE = 'None'
```

Comment:
What is `DCS_SESSION_COOKIE_SAMESITE`?

Reply:
DCS stands for django-cookies-samesite. This is the module we're using to add SAMESITE headers to the session cookie. Django introduced support for this in 2.0, but it's not complete. We can use django's functionality once we're on 3.1, which includes this fix: https://github.com/django/django/commit/b33bfc383935cd26e19a2cf71d066ac6edd1425f

For now, we'll need to continue to use the django-cookies-samesite package, which requires this setting name change to not conflict with Django's setting. Described here: https://pypi.org/project/django-cookies-samesite/

Reply:
Great. Thanks for the explanation.